File: spec/code_samples/Python requests/wallets/get.py
import requests

url = "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/wallets"

querystring = {"exchange":"SOME_STRING_VALUE","keyId":"SOME_STRING_VALUE"}

response = requests.request("GET", url, params=querystring)

print(response.text)

File: spec/code_samples/Php http2/wallets/get.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/wallets');
$request->setRequestMethod('GET');
$request->setQuery(new http\QueryString(array(
  'exchange' => 'SOME_STRING_VALUE',
  'keyId' => 'SOME_STRING_VALUE'
)));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();